server/hotrod/src/test/resources/create_certs.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+echo "==== Removing old files ===="
+find . -name "*.jks" -exec rm -rf {} \;
+
+echo "==== Generating private keys ===="
+keytool -genkey -noprompt -trustcacerts -keyalg RSA -alias "default" -dname "CN=HotRod_1, OU=Infinispan, O=JBoss, L=Red Hat, ST=World, C=WW" -keypass "secret" -storepass "secret" -keypass "secret2" -keystore "password_server_keystore.jks"
```

Comment:
I have a mixed feelings about it. On one hand - yes, this test would be really descriptive but on the other hand we should test Infinispan bits not OpenJDK SSL support. 

Anyway, I created https://issues.jboss.org/browse/ISPN-6541

@vjuranek any thoughts?